project/ShangHungWan/README.md

```diff
@@ -0,0 +1,121 @@
+# rtreelib: A C++ R-tree Library
+
+This is an R-tree library which implemented with C++ to achieve the performance. If you don't know what's R-tree, you can refer to [wiki](https://en.wikipedia.org/wiki/R-tree) [2] or original paper (Antonin Guttman. 1984. R-trees: a dynamic index structure for spatial searching. SIGMOD Rec. 14, 2 (June 1984), 47–57. <https://doi.org/10.1145/971697.602266>).
```

Comment:
Please use reasonable line width (e.g., 79 chars) so that I can add comments at which I want

Reply:
Changed the line width to 80 chars.